lib/ain-evm/src/precompiles/token_split.rs

```diff
@@ -0,0 +1,220 @@
+use std::collections::BTreeMap;
+
+use ain_contracts::{dst20_address_from_token_id, validate_split_tokens_input, TokenSplitParams};
+use ain_cpp_imports::{bridge::ffi::TokenAmount, split_tokens_from_evm};
```

Comment:
```suggestion
use ain_cpp_imports::{TokenAmount, split_tokens_from_evm};
```

Reply:
Resolved in [7340314](https://github.com/DeFiCh/ain/pull/2850/commits/7340314660205f1a7c8d0b85a863d404d7975243)